script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -357,17 +357,17 @@ The officers raise their pistols and fire in unison.
 [sha:b2e0bbb2bbbff3b8219417947d286c828a48ad33]{
 -- Page 400, Offset 9636.
 -- だがヤツは微動だにしない。
-However, the man does not budge.
+However, he doesn't so much as lift a finger.
```

Comment:
Not sure I love this because it implies action done, when I think the idea here is more "he got shot but just fucking tanked it". Doesn't budge, doesn't even blink/flinch